setup.sh

```diff
@@ -375,6 +375,12 @@ function _post() {
         echo_info "You can now use the box command to manage swizzin features, e.g. \`box install nginx panel\`"
     fi
     echo_docs getting-started/box-basics
+    #
+    # Run the bash_completion installer from the update folder
+    if ! bash /etc/swizzin/scripts/update/bash_completion.sh; then
```

Comment:
As it is now, this will always return true.

As far as I can see, there's nothing inside of `bash_completion.sh` in the likes of `exit 1` which would then send the escape code back for the `bash` call to return. As it is right now, if there's a failure in that update script, this condition will not trigger.

I'm also wondering what honestly _could_ fail, and I don't see that much... might as well remove the if statement here in my opinion.

Reply:
This is just pulled over from the dependencies example which I pointed userdocs to in order to help get the PR as this snippet falls somewhere between function and snippet. You *could* set `-e` on the completion script and then if any command errors at all the script will exit. Is this the behavior we want though?

Reply:
aha, I see. If the bash_completions fail, I don't think the whole installer should exit out.

Reply:
Since this is running on a bash subshell (either run the command as `bash -e ...` or `set -e` at the top of the snippet), only the running shell should exit

Reply:
Oh, sorry, I was referring to the exit two lines below this one (L382). That one would exit the whole `setup.sh`

Reply:
It was from this, the only other example of this behaviour.

https://github.com/swizzin/swizzin/blob/master/setup.sh#L205-L208

Reply:
Ah I see, the `apt` functions there are the ones that will throw an `exit 1`, which is then forwarded back to the `bash` on those lines, which will then throw that echo and kill the script.